assets/src/components/FeaturesTable.js

```diff
@@ -292,10 +292,12 @@ export default class FeaturesTable extends HTMLElement {
              * When the user has selected an item and highlighted it
              * @event features.table.item.highlighted
              * @property {string} itemFeatureId The feature ID of the selected item
+             * @property {string} itemLayerId The layer ID of the selected item
              */
             mainEventDispatcher.dispatch({
                 type: 'features.table.item.highlighted',
                 itemFeatureId: activeItemFeatureId,
+                itemLayerId: this.layerId,
             });
```

Comment:
I would also always give the element ID, just in case (we can then retrieve its properties, etc.)
 
```suggestion
            mainEventDispatcher.dispatch({
                type: 'features.table.item.highlighted',
                elementId: this.id,
                itemFeatureId: activeItemFeatureId,
                itemLayerId: this.layerId,
            });
```

@Gustry please also add the `layerId` and the element ID it in the event `features.table.item.dragged`